### ISSUE TITLE
Discussion of the data quality

### ISSUE THREAD

**Issue by baolenmeng** [ShichenXie/pedquant#6]
Hi ShichenXie,
Thank you for providing the package!
I saw a discussion of the data quality among pedquant, tushare, and joinquant:
https://zhuanlan.zhihu.com/p/266430753?
See Section 3.
Have you solved it?

**Comment by baolenmeng** [comment]
One follow-up question: could I use the function md_stock to download S&P 500 market index?

**Comment by ShichenXie** [comment]
1. The issue discussed in that post should be solved. You can upgrade to the latest version and try again. 
2. The inner function md_stooq provide an interface to query data from https://stooq.com/, which including global stock  indices. `dtsp = pedquant:::md_stooq('^SPX')`

**Comment by baolenmeng** [comment]
Sorry for my stupid question: is md_stooq an inner function in related R package?
I install and library the pedquant package. But commands `?md_stooq`, and `??md_stooq` return nothing.

**Comment by ShichenXie** [comment]
你先升级到最新版本0.1.9，然后再运行 `dtsp = pedquant:::md_stooq('^SPX')`，这个函数的参数和md_stock的一致的。

**Comment by baolenmeng** [comment]
Thank you for the reply. I have upgraded `pedquant` to v 0.1.9 and I also upgraded my R and Rstudio to the newest versions. However, once I run `md_stooq`, Rstudio always reported: "R encountered a fatal error. The session was terminated." I'll try later again. From the Console window, I judge the data is downloaded, since the window shows: `1/1 spx`. Then Console leaves a few blank lines and the error message pops up.
